people/puna/milly_or_polly.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="css/main.css" />
+    </head>
+    <body>
+    <div style="text-align: center;">
+<h1 id="name" style="text-align: center;">CHOOSE WISELY</h1>
+<h2 style="text-align: center;"><span id="content"> Who do you love more? Polly or Milly? </span></h2>
+</div>
+        <div class = "invoca"><a class="invocaLink"> <img src="images/Milly.jpg" alt="Milly" /></a>&nbsp;&nbsp;&nbsp;<img src="images/Polly.jpg" alt="Polly" />
+		</div>
+            </a>
+          
+
+<p>
+
+        <script type="text/javascript">
+         var polly = prompt("How many times have you petted Polly today?");
+		var milly = prompt("How many times have you petted Milly today?");
+
+		if (polly > milly) {
```

Comment:
So you're looking for how to ... _transform_ their _rotation_ huh? Good luck! You're free to merge after you can rotate them!